modules/creating-a-custom-image-from-default-image.adoc

```diff
@@ -10,6 +10,11 @@ You can create a custom image by adding OS packages or applications to a default
 .Prerequisites
 
 * You know which default image you want to use as the base for your custom image. 
++
+[IMPORTANT]
+====
+If you want to build a custom Elyra image, you must customize an existing {productname-short} Elyra image.
```

Comment:
Im thinking more about highlighting that you need to edit it directly. What you think of this? 
```suggestion
If you want to customize an Elyra image, you must modify an existing {productname-short} Elyra image.  
```